plugins/src/webrtcsink/imp.rs

```diff
@@ -362,7 +360,8 @@ fn setup_encoding(
             // to incorrect color display in Chrome (but interestingly not in
             // Firefox). In any case, restrict to exclude RGB formats altogether,
             // and let videoconvert do the conversion properly if needed.
-            structure_builder = structure_builder.field("format", &gst::List::new(&[&"NV12", &"YV12", &"I420"]));
+            structure_builder =
+                structure_builder.field("format", &gst::List::new(&[&"NV12", &"YV12", &"I420"]));
```

Comment:
Why all the `&` :sob:
They'are all not needed and just add line-noise

Reply:
not @thiblahute's fault, probably just me writing this code as it once needed to be written?

Reply:
Yeah, that was needed until 2-3 months ago